### PR TITLE
Frontend Changes

### PR DESCRIPTION
**Pull request by mehedi-rul**
Modified:
- Login page changed to logo,text and button color and logo background color

Added:
- Active poll, Archived poll, manage poll field has been added to the side navbar on Dashboard.
- Added Dynamic Iframe Active poll, Archived poll.
- Active poll, Archived poll, manage poll activities has been added new icon.
- Addad a new field in the user section table and new user and edit user page, its name is Agency. And customized fields FirstName and LastName to Name.
- Added a new field in the edit user page, its name is Active.
- Addad a new manage poll section table with new user and edit user.

### CODE REVIEW

**Comment by mehedi-rul** [comment]
v1.5.5 Active poll, Archived poll, manage poll update

File: src/views/managePoll.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      is_active: false,
+    };
+  },
+  watch: {
+    pollsUrl() {
+      this.resourceUrl = this.pollsUrl;
+    },
+    data() {
+      if (this.data && !this.data.id) {
+        this.data.id = this.data.id;
```
Comment (paulobernardoaf) [inline src/views/managePoll.vue:70]: Is this really correct?

File: src/views/User.vue [hunk 2]
```diff
@@ -122,6 +137,16 @@ export default {
         return newRole;
       },
     },
+    isActive: {
+      get() {
+        console.log(this.data.is_active);
```
Comment (paulobernardoaf) [inline src/views/User.vue:142]: Please remove console

**Review by paulobernardoaf — CHANGES_REQUESTED** [review]
Run `yarn lint` after making changes

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,56 +1,17 @@
...
-v1.4.0
-
-Added:
-- Poll Dashboard using Power Bi
-
-Fixed:
-- Incorrect incoming and outgoing labels
-
-v1.3.2
+release v1.5.5
```
[the remark this answers is not in the document]
Comment (mehedi-rul) [reply CHANGELOG.md]: done

**Comment by mehedi-rul** [comment]
This issue has been resolved

**Review by paulobernardoaf — APPROVED** [review]

**Pull request merged**